lib/plugins/jsonBodyParser.js

```diff
@@ -28,8 +28,13 @@ function jsonBodyParser(options) {
         // save original body on req.rawBody and req._body
         req.rawBody = req._body = req.body;
 
-        if (req.getContentType() !== 'application/json' || !req.body) {
-            return next();
+        var jsonPatternMatcher = new RegExp('^application/[a-zA-Z.]+\\+json');
```

Comment:
Since we're already doing this match here before we call into it jsonBodyParser, we can remove this content type check entirely.